nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/notification/SendMail.java

```diff
@@ -200,6 +200,12 @@ protected void send(DocumentModel doc)
         }
     }
 
+    @VisibleForTesting
```

Comment:
Please use just a comment instead and don't import an annotation from a third-party library for this.